OWML.Launcher/App.cs

```diff
@@ -70,6 +72,64 @@ private void LocateGamePath()
             }
         }
 
+        private void CheckGameVersion()
```

Comment:
This is too low level for App.cs, can you move it to another/new class?

Reply:
well, I can carry it to versionReader

Reply:
C# has version comparison functionality built-in: https://www.c-sharpcorner.com/blogs/comparing-version-number-in-c-sharp
Use that and your code will be simple enough to stay where it is.